app/src/main/java/com/jetbrains/handson/mpp/mobile/MainActivity.kt

```diff
@@ -16,6 +19,12 @@ class MainActivity : AppCompatActivity(), ApplicationContract.View {
 
     private lateinit var departures: MutableList<DepartureInformation>
 
+    private var numAdults = 1
```

Comment:
I've implemented this by making the onActivityResult function able to access the presenter. Is it bad practice to declare the presenter in this way?

Reply:
Not at all, actually it's pretty standard to have the presenter declared as a lateinit var so that it can be accessed from different functions within the view 🙂